Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1330,6 +1330,28 @@ func _vjpSelu<T: TensorFlowFloatingPoint>(
     })
 }
 
+/// Returns a tensor by applying the swish activation function, namely
+/// `x * sigmoid(x)`.
+///
+/// Source: "Searching for Activation Functions" (Ramachandran et al. 2017)
+/// https://arxiv.org/abs/1710.05941
+@inlinable
+@differentiable
+public func swish<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    x * sigmoid(x)
+}
+
+@inlinable
+@derivative(of: swish)
+func _vjpSwish<T: TensorFlowFloatingPoint>(
```

Comment:
We shouldn't need a custom vjp at all here. The compiler should be able to synthesize this automatically! Can you remove it?

Reply:
Ah, I see a justification for the custom vjp at [the link you posted](https://github.com/tensorflow/tensorflow/blob/7d9144dc5f7725f9b4fbf005234d95fa7ded4baa/tensorflow/python/ops/nn_impl.py#L521).

That's a good justification, and this custom vjp does do the same thing.

You should add a comment explaining why there is a custom vjp.